Tools/AP_Periph/can.cpp

```diff
@@ -1348,6 +1376,42 @@ static void processRx(void)
     }
 }
 
+#if HAL_PERIPH_CAN_MIRROR
+static void processMirror(void)
```

Comment:
I'd like this to be in can_mirror.cpp and have it as a static function on the AP_Periph_FW class

Reply:
Only possible if we move the `instance_t` struct into the class. There's less out of scope change then there used to be if I do that now.